### ISSUE TITLE
Clicking outside the box does not right away close the magic tag drop down

### ISSUE THREAD

**Issue by paaljoachim** [CalderaWP/Caldera-Forms#877]
I have been using the new magic tag drop down and really like it!
But clicking outside the box it kinda gets stuck in an open position and after a bit of clicking elsewhere it closes.

![screen shot 2016-10-12 at 14 12 32](https://cloud.githubusercontent.com/assets/5323259/19322193/121f5b56-9086-11e6-87df-9aa1403a0828.png)

**Comment by Shelob9** [comment]
@paaljoachim Is this only happening in the message field of auto-responder, or is it happening in all fields?

Also, what OS and browser is this happening in?

**Comment by paaljoachim** [comment]
This is in Chrome on a Mac OS 10.11. (It is probably Chrome acting up.)

Clicking various fields in Email. I just tested again. It can get a little stuck. Below I opened the BSS field and then the Email Recipients. Both are open at the same time.

![screen shot 2016-10-12 at 21 37 20](https://cloud.githubusercontent.com/assets/5323259/19333486/3c4cb3c2-90c4-11e6-8ed0-30e2ae2339bc.png)

Auto Responder seems to be working alright.
